docs/resources/aws_ec2_instances.md.erb

```diff
@@ -66,14 +75,15 @@ Provides a list of the instance ids that were found in the query.
 
 ## Matchers
 
-For a full list of available matchers, please visit our [Universal Matchers page](https://www.inspec.io/docs/reference/matchers/). 
+For a full list of available matchers, please visit our [Universal Matchers page](https://www.inspec.io/docs/reference/matchers/).
 
 ### exist
 
 The control will pass if the filter returns at least one result. Use `should_not` if you expect zero matches.
 
-    # Verify that at least one EC2 Instance exists.
-    describe aws_ec2_instances
-      it { should exist }
-    end   
-
+``` 
```

Comment:
I notice we use the indention method in other places. I personally prefer this approach, but should we add `ruby` to the end?